minio/policy.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
rename the file to `public-policy.json` or similar to communicate what it does?

Reply:
done